src/websocketHelpers.js

```diff
@@ -0,0 +1,114 @@
+const { DateTime } = require('luxon');
```

Comment:
Why luxon and not moment?

Reply:
you could also ask why not [date-fns](https://github.com/date-fns/date-fns)  😄 

just personal preference, but here's a good rundown: https://moment.github.io/luxon/docs/manual/why.html

essentially it's newer, less bloat, newer js features, and most importantly, immutable.